dotCMS/src/main/java/com/dotcms/security/secret/ServiceIntegrationBean.java

```diff
@@ -0,0 +1,74 @@
+package com.dotcms.security.secret;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ServiceIntegrationBean implements Serializable {
+
+    private String serviceKey;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'serviceKey' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4623512)